doc_src/tutorial.hdr

```diff
@@ -581,6 +581,16 @@ To prepend /usr/local/bin and /usr/sbin to `$PATH`, you can write:
 >_ set PATH /usr/local/bin /usr/sbin $PATH
 \endfish
 
+To remove /usr/local/bin from `$PATH`, you should first find its index in the list, you can write:
+
+\fish{cli-dark}
+>_ echo $PATH
+> /home/hunsu/bin/ /usr/bin/ /bin /home/hunsu/Android/Sdk/tools /home/hunsu/Android/Sdk/platform-tools /usr/local/sbin /usr/local/bin
+> # faster way to find the index
+> echo $PATH | tr " " "\n" | nl"
```

Comment:
This line has a trailing quote - `nl"`.

Also, it'd break on $PATH elements with spaces in them.

Instead, I'd recommend `string escape -- $PATH` - that will display one element per line in a way that is unambiguous (e.g. `\n` instead of a literal newline).

Also, `contains -i` is useful here.